Source/WebCore/platform/network/soup/CookieJarSoup.cpp

```diff
@@ -270,26 +280,25 @@ void setCookies(const NetworkStorageSession& session, const Vector<String>& cook
     // To prevent from calling cookie change handlers during it
     // blocking all the handlers and then unblocking after the cookies are set
     guint signal_id = g_signal_lookup("changed", soup_cookie_jar_get_type());
-    GArray* blocked_handler_id_array = g_array_new(FALSE, FALSE, sizeof(guint));
+    GArray* blocked_handler_id_array = g_array_new(FALSE, FALSE, sizeof(gulong));
```

Comment:
I would probably do this in a separate patch because it's an unrelated change, but that's up to you.